lib/occi/cli/helpers/create_helper.rb

```diff
@@ -43,8 +45,24 @@ def helper_create_resource(options)
     #res.check
 
     Occi::Cli::Log.debug "Creating #{options.resource.inspect}: #{res.inspect}"
+    new_link = create(res)
+    helper_create_wait(new_link, options.wait_for_active) if options.wait_for_active > 0
+
+    new_link
+  end
 
-    create res
+  def helper_create_wait(resource_link, timeout)
+    begin
+      Timeout::timeout(timeout) {
+        while true
+          desrc = describe(resource_link).first
+          break if !desrc.respond_to?(:state) || desrc.state == 'active'
+          sleep MIN_TIMEOUT
+        end
+      }
+    rescue Timeout::Error
+      Occi::Cli::Log.warn "Attempted to wait #{timeout}s for #{resource_link.inspect} to become active"
```

Comment:
Perhaps also log something if the wait is successful? Either on entry into the wait loop, or on break? May help us catch lag complaints in the future.